app/models/pydantic/versions.py

```diff
@@ -59,7 +60,14 @@ class VersionUpdateIn(StrictBaseModel):
 
 class VersionAppendIn(StrictBaseModel):
     source_uri: List[str]
-
+    source_driver: Optional[VectorDrivers] = Field(
+        None, description="Driver of source file. Must be an OGR driver"
+    )
+    layers: Optional[List[str]] = Field(
+        None,
+        description="List of layer names to append to version. "
+        "If not set, all layers in source_uri will be appended.",
+    )
```

Comment:
It looks to me like if layers is not specified it will be assumed that there is only a layer named like the file (though I may be looking in the wrong place in the code): https://github.com/wri/gfw-data-api/blob/master/app/tasks/vector_source_assets.py#L209-L214

Reply:
Good catch, I corrected the description. Layer names are required for .GDB and .GPKG, otherwise the file name is used as the layer name.